sdk/logs/src/main/java/io/opentelemetry/sdk/logs/internal/SdkLoggerProviderUtil.java

```diff
@@ -55,4 +56,20 @@ public static void addLoggerConfiguratorCondition(
           "Error calling addLoggerConfiguratorCondition on SdkLoggerProviderBuilder", e);
     }
   }
+
+  /** Reflectively set exception attribute resolver to the {@link SdkLoggerProviderBuilder}. */
+  public static void setExceptionAttributeResolver(
```

Comment:
What is the motivation for having this implemented reflectively, rather than using simple method call? Why can't `setExceptionAttributeResolver` be public?

Reply:
This is a pattern we use for experimental APIs. Given our strong backwards compatibility guarantees, we like to be conservative when introducing new APIs, and let them "bake" for a bit so we can solicit feedback and make breaking changes if needed. The package private setter + reflection is the trick to achieve that: it forces user call this utility method which is in an internal package.

Reply:
Got it, thanks!